getting-started/versioning.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Versioning"
+weight: 100
+---
+
+# Versioning
+
+### Format
+
+`v<major>.<minor>`
+
+examples: `v0.1` `v0.2`
+
+All components of this API have single, common version.
+
+### Endpoints
+
+Our components have different endpoints for every version. For example here is what url with version looks like for customer chat API:
```

Comment:
Our components have different endpoints for every version. For example here is what url with version looks like for customer chat API: -> Our components have different endpoints for every version. For example here is what a URL with a version looks like for the customer chat API: